### PR TITLE
Lecture 10: Sequential models

### PR DESCRIPTION
**Pull request by abhimohta**
https://abhimohta.github.io/pgm-spring-2019/notes/lecture-10/

### CODE REVIEW

**Comment by abhimohta** [comment]
When can we expect the review?

**Comment by alshedivat** [comment]
@zhisbug, can you take a look? Thanks!

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: use $\exp$ instead of raw text $exp$ in a math env.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md:78]: Make this left aligned using &.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: make this left aligned.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md:205]: This one looks great.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: left aligned.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: since the posterior covariance ($V_{1|2}$ above) and does not depend on the observed
-> since the posterior covariance ($V_{1|2}$ above) does not depend on the observed

an extra `and`?

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md:303]: Align at the `=` is also fine. Make sure your way of alignment is consistent across the notes.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: Are you sure the equations here are correct? It seems to me there is no connections between x and y from these equations...

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: Same here

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: typo: reffered

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md:525]: This alignment looks great.

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md:587]: what are m and V here?

File: _posts/2019-02-18-lecture-10.md
Comment (zhisbug) [inline _posts/2019-02-18-lecture-10.md]: f2 and f3 are not properly formatted here... Please fix.

**Review by zhisbug — CHANGES_REQUESTED** [review]
@abhimohta  @iedwardwangi  @alankarj 
Despite a few minor problems I have spotted, this is a great scribe note.
Please address my comments and I'll take a second review before merging.

**Comment by abhimohta** [comment]
@zhisbug the requested changes have been made. Can you please have a look?

**Comment by zhisbug** [comment]
LGTM, I will make the merge.

**Review by zhisbug — APPROVED** [review]

**Pull request merged**
